app/src/main/java/org/mozilla/fenix/settings/PairFragment.kt

```diff
@@ -32,7 +32,6 @@ import org.mozilla.fenix.ext.showToolbar
 class PairFragment : Fragment(R.layout.fragment_pair), UserInteractionHandler {
 
     private val qrFeature = ViewBoundFeatureWrapper<QrFeature>()
-    private val preferences = PreferenceManager.getDefaultSharedPreferences(context)
```

Comment:
Looks like the fix was to just change where we were accessing this? Could we just make this lazy? 🤔

Reply:
Or alternatively add a getter if it's about needing to `getDefaultSharedPreferences` every time?